packages/toast/toast-container.js

```diff
@@ -0,0 +1,138 @@
+                const remove = [];
+                for (const toast of this._toasts) {
+                    if (Date.now() <= toast[1].duration) keep.push(toast);
+                    else remove.push(toast);
+                }
+                // collapse toasts that will be removed
+                const collapseTasks = [];
+                for (const [id] of remove) {
+                    const el = this.renderRoot.querySelector(`#${id}`);
+                    collapseTasks.push(el.collapse());
+                }
+                // once all toasts that should be removed have been collapsed, remove them from the map
+                Promise.all(collapseTasks).then(() => {
+                    if (keep.length != this._toasts.size) this._toasts = new Map(keep);
+                });
+            }, 500);
+        }
+    }
+    
+    disconnectedCallback() {
+        super.disconnectedCallback();
+        if (this._interval) clearTimeout(this._interval);
+    }
+
+    static init() {
+        let el = document.querySelector('f-toast-container');
+        if (!el) {
+            el = document.createElement('f-toast-container');
+            document.body.appendChild(el);
+        }
+        return el;
+    }
+
+    get _toastsArray() {
+        return Array.from(this._toasts).map(([,toast]) => toast);
+    }
+
+    /**
+     * 
+     * @param {String|Number} id 
+     * @returns {ToastOptions}
+     */
+    get(id) {
+        if (!id) throw new Error('undefined "id" given when attempting to retrieve toast');
+        if (typeof id !== 'string' && !Number.isInteger(id)) throw new Error('"id" must be number or string when attempting to retrieve toast');
+        return this._toasts.get(id);
+    }
+
+    /**
+     * 
+     * @param {Object} toast 
+     * @returns {FabricToastContainer}
+     */
+    set(toast) {
+        if (!toast.id) throw new Error('invalid or undefined "id" on toast object');
+        const result = this._toasts.set(toast.id, {
+            ...toast,
+            duration: Date.now() + toast.duration || Number.POSITIVE_INFINITY,
+        });
+        this._toasts = new Map(Array.from(this._toasts));
+        return result;
+    }
+
+    /**
+     * 
+     * @param {String|Number} id
+     * @returns {ToastOptions}
+     */
+    async del(id) {
+        if (!id) throw new Error('undefined "id" given when attempting to retrieve toast');
+        if (typeof id !== 'string' && !Number.isInteger(id)) throw new Error('"id" must be number or string when attempting to retrieve toast');
+        const el = this.renderRoot.querySelector(`#${id}`);
+        await el.collapse();
+        const result = this._toasts.delete(id);
+        this._toasts = new Map(Array.from(this._toasts));
+        return result;
+    }
+
+    render() {
+        return html`
+            <link
+                rel="stylesheet"
+                type="text/css"
+                href="https://assets.finn.no/pkg/@fabric-ds/css/v1/fabric.min.css"
+            />
```

Comment:
As per discussions with @trygve-lie and @benja this sort of inline Fabric inclusion is not really what we want. We will be revisiting this in the near future.